src/views/research/l10n.json

```diff
@@ -0,0 +1,8 @@
+{
+    "research.title":"Research on Scratch",
+    "research.conductors":"Research on Scratch is being conducted by members of the MIT Scratch Team and researchers at other universities, including Yasmin Kafai at the University of Pennsylvania Graduate School of Education, Karen Brennan at the Harvard Graduate School of Education, Benjamin Mako Hill at the University of Washington, Andrés Monroy Hernandez at Microsoft Research, Mimi Ito and Crystle Martin at the University of California, Irvine, Quinn Burke at College of Charleston, Deborah Fields at Utah State University, and Kylie Peppler at Indiana University.",
+    "research.privacy":"By sharing projects and participating in the Scratch online community, you are helping us better understand how people can use and learn with Scratch. Any publicly shared projects, comments, or other material on the Scratch site may be included in the research analysis, presentations, papers, and reports. No personally identifiable information is shared. (If you have any questions, please use the <a href=\"/contact-us/\">Contact Us</a> form.)",
```

Comment:
Format strings can't have embedded HTML. It will need a placeholder in this string and an additional string for the contact link text. For example, `please use the {contactLink} form`, and `"research.contactLinkText": "Contact Us"`

We're also going to use the `blahLink`, `blahLinkText` convention from now on so that it's easy for translators to find the corresponding link text when they're translating a phrase that includes links.

Reply:
Got it.